libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,16 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+# rubocop:disable Documentation
+class Chef
+  # Making the sensitive attribute passive for older chef versions
+  class Resource
+    def sensitive(_ = nil)
```

Comment:
I went ahead and added the functionality and pushed the new commit (I can squash later if you want). As far as I can see it won't produce any issues and if we want to consume it elsewhere we won't have to worry about breaking compatibility.